iphone/cli/hooks/run.js

```diff
@@ -35,6 +35,7 @@ exports.init = function (logger, config, cli) {
 			const startLogTxt = __('Start simulator log'),
 				endLogTxt = __('End simulator log'),
 				levels = logger.getLevels(),
+				trimRE = new RegExp('^.*' + builder.tiapp.name + '\\[[^\\]]+\\]\\s*', 'g'),
```

Comment:
<!--
  0 failure: 
  1 warning:  iphone/cli/hooks/...
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/hooks/run.js;
  Line: 38;
-->

- :warning: iphone/cli/hooks/run.js line 38 – Found non-literal argument to RegExp Constructor (security/detect-non-literal-regexp)